runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersConfig.java

```diff
@@ -33,6 +33,17 @@
 {
 	@ConfigItem(
 		position = 0,
+		keyName = "showHomeMinigameTeleports",
+		name = "Teleport cooldown timers",
+		description = "Configures whether timers for home and minigame teleport cooldowns are displayed"
+	)
+	default boolean showHomeMinigameTeleports()
```

Comment:
Any reason why you couldn't just append this to end?

Reply:
It's a timer that I'd think most people would want to notice more than the others listed, but I don't have a strong opinion on where it should be placed.